src/main/java/com/google/cloud/tools/appengine/AppEngineDescriptor.java

```diff
@@ -50,6 +51,9 @@ private AppEngineDescriptor(Document document) {
    * @throws SAXException malformed XML
    */
   public static AppEngineDescriptor parse(InputStream in) throws IOException, SAXException {
+    if (in == null) {
+      throw new IOException("Null input");
```

Comment:
Is IOException right?, given this is a programmer error, not an IO system error.

The nullaway plugin assumes without a `@Nullable` annotation, the parameter must not be null. So the person calling `parse` is responsible for the null check, or errorprone-nullaway will error out.